argus/backend/service/testrun.py

```diff
@@ -307,23 +307,27 @@ def get_run_events(self, run_id: UUID):
         }
         return response
 
-    def fetch_results(self, test_id: UUID, run_id: UUID) -> dict:
-        query_fields = ["column", "row", "value", "status"]
+    def fetch_results(self, test_id: UUID, run_id: UUID) -> list[dict]:
+        session = ScyllaCluster.get_session()
```

Comment:
you should probably use `ScyllaCluster.prepare` to get a `PreparedStatement` statement instead

Reply:
true, fixed, please take  a look again

Reply:
Ah, I meant this [helper method](https://github.com/scylladb/argus/blob/master/argus/backend/db.py#L70-L75) as it also saves this statement so you don't have to on every query.

Reply:
ok, I see, fixed.